### PR TITLE
update reporting form to migrate rep when the forking market is selec…

### LINKED ISSUE #3980: Forking: Migration Form locked up REP
detect that user has locked up REP and display notice. `state.loginAccount.reporting` has a list of all contracts user has staked on, if there is a totalAmount for reporting,disputing or participation tokens show notice.

https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=2111%3A889

image

### PR DESCRIPTION
**Pull request by bthaile**
…ted to report on.

closes https://github.com/augurproject/augur/issues/3980

next iteration in migrate rep form. Users will be able to migrate rep and report and migrate markets. 

markets still in pre-reporting can't be migrated until we figure out an error occuring on contract call.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🍦

 This link will always point to this PR's latest changes: https://peek.run/qo0h6wo

File: packages/augur-ui/src/modules/modal/reporting.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import {
   SCALAR,
   REPORTING_STATE,
   INVALID_OUTCOME_NAME,
+  ZERO,
```
Comment (adrake33) [inline packages/augur-ui/src/modules/modal/reporting.tsx]: It doesn't look like this is being used -- can probably be removed.

File: packages/augur-ui/src/modules/modal/reporting.tsx [hunk 2]
```diff
@@ -24,8 +25,7 @@ import {
 } from 'modules/contracts/actions/contractCalls';
 
 import Styles from 'modules/modal/modal.styles.less';
-import { createBigNumber } from 'utils/create-big-number';
-import { convertDisplayValuetoAttoValue, Getters } from '@augurproject/sdk';
+import { Getters } from '@augurproject/sdk';
```
Comment (adrake33) [inline packages/augur-ui/src/modules/modal/reporting.tsx:26]: Same here.

**Comment by bthaile** [comment]
addressed comments

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 3]
```diff
@@ -588,52 +610,76 @@ export class ReportingBondsView extends Component<
...
-          preFilledStake={preFilledStake}
-          stakeError={stakeError}
-          threshold={userAvailableRep}
-        />
+        {migrateRep && createBigNumber(inputtedReportingStake).lt(createBigNumber(userAvailableRep)) &&
+          <DismissableNotice
+            show={true}
+            description=''
+            title='Are you sure you only want to migrate a portion of your REP to this universe?
+            If not, Go back and select the ‘MAx’ button underneath the amount input to migrate your Full REP Amount.'
```
Comment (adrake33) [inline packages/augur-ui/src/modules/reporting/common.tsx]: Nit - capitalization: `If not, go back and select the ‘Max’ button underneath the amount input to migrate your full REP amount.`

File: packages/augur-ui/src/modules/modal/reporting.tsx (hunk 2 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/modal/reporting.tsx:26]: There was a format not used Getters is used for putting in `Getters.Accounts.UserCurrentOutcomeDisputeStake`

**Review by adrake33 — APPROVED** [review]
LGTM. Recommend doing more testing later, as you mentioned.

File: packages/augur-ui/src/modules/reporting/common.tsx (hunk 3 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/reporting/common.tsx]: good one, will fix

**Comment by bthaile** [comment]
pushed changes for capitalization and cleaned up sentence.

**Pull request merged**